components/unit-cards/unit-squad-card.tsx

```diff
@@ -43,6 +43,11 @@ type UnitSquadInput = {
   };
 };
 
+const unfiformIconSize = {
+  width: 24,
+  height: 24,
```

Comment:
Let's make it little bit smaller 20 / or 22.

Reply:
done